Makefile

```diff
@@ -128,7 +128,13 @@ test: codegen fmt vet envtest ## Run tests.
 
 .PHONY: test-e2e
 test-e2e: codegen fmt vet envtest ## Run e2e tests.
-	GOFLAGS="-count=1" go test -timeout 20m -v ./tests/e2e/... 
+	GOFLAGS="-count=1" go test -timeout 20m -v ./tests/e2e/...
```

Comment:
Actually, I'm kind of thinking maybe `make test-e2e` doesn't make sense to run anymore and I'm wondering if we really need it. Some of the tests require an environment variable; others don't. It would be weird to set the environment variable to "progressive" for example and run this, in which the "progressive" version of the functional test executes and so does the PPND test.